scripts/add_ts_data_bidmatrixinfo.py

```diff
@@ -0,0 +1,59 @@
+###this script adds time series data to the bid matrix information objects in the powerops client
+
+from cognite.client.data_classes import TimeSeries
+from cognite.powerops import PowerOpsClient
+import random
+import datetime
+
+
+power = PowerOpsClient.from_settings()
+cdf = power.cdf
+
+start_time = int((datetime.datetime.now() - datetime.timedelta(days=365)).timestamp() * 1000)
+end_time = int(datetime.datetime.now().timestamp() * 1000)
+
+# creating time series, with data points every hour
+if not cdf.time_series.retrieve(external_id="wim_ts_hourly"):
+    wim_ts_hourly = cdf.time_series.create(TimeSeries(name="wim_ts_hourly", external_id="wim_ts_hourly"))
+    data_points = [
+        {"timestamp": start_time + i * 3600 * 1000, "value": random.random()}
```

Comment:
**question:** does this create a datapoint at every whole hour? have we checked with @notrixbe to see if the datapoints produced here match the constraints needed?